views/partials/account-sidebar.ejs

```diff
@@ -11,12 +11,10 @@
     <div id="user-country"><%-profileData.country%></div>
   </div>
   <div class="sidebar-all-tabs">
-    <div class="sidebar-tab">Change credentials</div>
-    <div class="sidebar-tab">
-      <a href="/account/change-profile">Edit Profile</a>
-    </div>
-    <div class="sidebar-tab">Play sessions</div>
-    <div class="sidebar-tab">Detailed stats</div>
-    <div class="sidebar-tab">Customize site</div>
+    <a class="sidebar-tab">Change credentials</a>
```

Comment:
It's kind of silly but on mobile it's no longer a sidebar hence the class name doesn't really make much sense.